src/Git/Git.php

```diff
@@ -240,8 +240,8 @@ protected function getCommandContext($paths, $message)
         return [
             'paths' => collect($paths)->implode(' '),
             'message' => $message,
```

Comment:
Why not escape the message and the paths as well?

Reply:
Those aren't user input.

Reply:
Ok, but if you have a file with a quote in its name, they may cause unintended interpretation by the shell before the `git` commands gets to see them?

Reply:
Additionally, the commit messages appear to be generated by code and can be translated, such as e.g. here:

https://github.com/statamic/cms/blob/7911d611c737c4d2c3edc8bc9e03c4ab153d1a4d/src/Events/AssetDeleted.php#L16

Without escaping those messages here, everyone that writes those messages in the code or in the translations has to be aware that they will be included in shell commands and thus have to remember to always "pre-escape" them.

Reply:
Fair points, we can escape these too.

Reply:
Commit message is now escaped. Paths aren't, but they're only ever going to be what's defined in `config/statamic/git.php`. If people are adding paths in there that contain quotes, we can address that separately.